knimin/tests/test_ag_results_ready.py

```diff
@@ -25,13 +27,29 @@ def test_post(self):
         msg = 'Successfully updated barcodes to results ready status.'
         self.assertIn(msg, response.body)
 
+        os.environ["ASYNC_TEST_TIMEOUT"] = "30"
         with db._con.cursor() as cur:
             cur.execute("""SELECT results_ready
                            FROM ag.ag_kit_barcodes
                            WHERE barcode='000023299'""")
             result = cur.fetchone()[0]
             self.assertEqual(result, 'Y')
 
+        # test that error is raised if pdf directory does not exist
+        db.config.base_data_dir = 'phantasy_path'
+        response = self.post('/update_ready/', {})
+        self.assertEqual(response.code, 500)
```

Comment:
ideally we never 500, although it does seem appropriate in this situation. what do you think about presenting the user with a message about a bad configuration?